{
 "archived": false,
 "branch": "main",
 "conda-forge.yml": {},
 "feedstock_name": "exceptiongroup",
 "hash_type": "sha256",
 "linux_64_meta_yaml": {
  "about": {
   "dev_url": "https://github.com/agronholm/exceptiongroup",
   "doc_url": "https://docs.python.org/3/library/exceptions.html",
   "home": "https://pypi.org/project/exceptiongroup/",
   "license": "BSD-3-Clause",
   "license_family": "BSD",
   "license_file": "LICENSE",
   "summary": "Backport of PEP 654 (exception groups)"
  },
  "build": {
   "noarch": "python",
   "number": "0",
   "script": "/usr/share/miniconda3/envs/cf-scripts/conda-bld/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/bin/python -m pip install . -vv"
  },
  "extra": {
   "recipe-maintainers": [
    "xylar"
   ]
  },
  "package": {
   "name": "exceptiongroup",
   "version": "1.1.1"
  },
  "requirements": {
   "host": [
    "python >=3.7",
    "pip",
    "flit-scm"
   ],
   "run": [
    "python >=3.7"
   ]
  },
  "source": {
   "sha256": "d484c3090ba2889ae2928419117447a14daf3c1231d5e30d0aae34f354f01785",
   "url": "https://pypi.io/packages/source/e/exceptiongroup/exceptiongroup-1.1.1.tar.gz"
  },
  "test": {
   "commands": [
    "pip check"
   ],
   "imports": [
    "exceptiongroup"
   ],
   "requires": [
    "pip"
   ]
  }
 },
 "linux_64_requirements": {
  "build": {
   "__set__": true,
   "elements": []
  },
  "host": {
   "__set__": true,
   "elements": [
    "flit-scm",
    "pip",
    "python"
   ]
  },
  "run": {
   "__set__": true,
   "elements": [
    "python"
   ]
  },
  "test": {
   "__set__": true,
   "elements": [
    "pip"
   ]
  }
 },
 "meta_yaml": {
  "about": {
   "dev_url": "https://github.com/agronholm/exceptiongroup",
   "doc_url": "https://docs.python.org/3/library/exceptions.html",
   "home": "https://pypi.org/project/exceptiongroup/",
   "license": "BSD-3-Clause",
   "license_family": "BSD",
   "license_file": "LICENSE",
   "summary": "Backport of PEP 654 (exception groups)"
  },
  "build": {
   "noarch": "python",
   "number": "0",
   "script": "/usr/share/miniconda3/envs/cf-scripts/conda-bld/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/bin/python -m pip install . -vv"
  },
  "extra": {
   "recipe-maintainers": [
    "xylar"
   ]
  },
  "package": {
   "name": "exceptiongroup",
   "version": "1.1.1"
  },
  "requirements": {
   "host": [
    "python >=3.7",
    "pip",
    "flit-scm"
   ],
   "run": [
    "python >=3.7"
   ]
  },
  "source": {
   "sha256": "d484c3090ba2889ae2928419117447a14daf3c1231d5e30d0aae34f354f01785",
   "url": "https://pypi.io/packages/source/e/exceptiongroup/exceptiongroup-1.1.1.tar.gz"
  },
  "test": {
   "commands": [
    "pip check"
   ],
   "imports": [
    "exceptiongroup"
   ],
   "requires": [
    "pip"
   ]
  }
 },
 "name": "exceptiongroup",
 "outputs_names": {
  "__set__": true,
  "elements": [
   "exceptiongroup"
  ]
 },
 "parsing_error": false,
 "pr_info": {
  "__lazy_json__": "pr_info/exceptiongroup.json"
 },
 "raw_meta_yaml": "{% set name = \"exceptiongroup\" %}\n{% set version = \"1.1.1\" %}\n\npackage:\n  name: {{ name|lower }}\n  version: {{ version }}\n\nsource:\n  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/exceptiongroup-{{ version }}.tar.gz\n  sha256: d484c3090ba2889ae2928419117447a14daf3c1231d5e30d0aae34f354f01785\n\nbuild:\n  noarch: python\n  script: {{ PYTHON }} -m pip install . -vv\n  number: 0\n\nrequirements:\n  host:\n    - python >=3.7\n    - pip\n    - flit-scm\n  run:\n    - python >=3.7\n\ntest:\n  imports:\n    - exceptiongroup\n  commands:\n    - pip check\n  requires:\n    - pip\n\nabout:\n  home: https://pypi.org/project/exceptiongroup/\n  summary: Backport of PEP 654 (exception groups)\n  license: BSD-3-Clause\n  license_family: BSD\n  license_file: LICENSE\n  dev_url: https://github.com/agronholm/exceptiongroup\n  doc_url: https://docs.python.org/3/library/exceptions.html\n\nextra:\n  recipe-maintainers:\n    - xylar\n",
 "req": {
  "__set__": true,
  "elements": [
   "flit-scm",
   "pip",
   "python"
  ]
 },
 "requirements": {
  "build": {
   "__set__": true,
   "elements": []
  },
  "host": {
   "__set__": true,
   "elements": [
    "flit-scm",
    "pip",
    "python"
   ]
  },
  "run": {
   "__set__": true,
   "elements": [
    "python"
   ]
  },
  "test": {
   "__set__": true,
   "elements": [
    "pip"
   ]
  }
 },
 "strong_exports": false,
 "total_requirements": {
  "build": {
   "__set__": true,
   "elements": []
  },
  "host": {
   "__set__": true,
   "elements": [
    "flit-scm",
    "pip",
    "python >=3.7"
   ]
  },
  "run": {
   "__set__": true,
   "elements": [
    "python >=3.7"
   ]
  },
  "test": {
   "__set__": true,
   "elements": [
    "pip"
   ]
  }
 },
 "url": "https://pypi.io/packages/source/e/exceptiongroup/exceptiongroup-1.1.1.tar.gz",
 "version": "1.1.1",
 "version_pr_info": {
  "__lazy_json__": "version_pr_info/exceptiongroup.json"
 }
}